package.json

```diff
@@ -176,7 +176,7 @@
   },
   "peerDependencies": {
     "@material-ui/core": ">=4.12",
-    "cozy-client": ">=27.19.1",
+    "cozy-client": ">=28.1.0",
```

Comment:
I agree, this is a BC.

Reply:
donc on doit avoir `BREAKING CHANGE: bla bla bla` dans **ce** commit message et non l'autre

Reply:
Le message "This update is necessary to use the `getSharingLink` function" va apparaître dans le message de BC dans le changelog de cozy-ui. A mon avis ça n'est pas très utile.  Je ne sais pas si c'est possible de ne pas commencer par le message de BC et d'avoir le BC quand même...

Dans le doute j'écrirai ça : 

BREAKING CHANGE: You need to update `cozy-client` to `>28.1.0` (the update was necessary to use the `getSharingLink` function)